### PR TITLE
Error message if login fails for db connection

### PR DESCRIPTION
**Pull request by schittarath3-rti**

### CODE REVIEW

**Comment by AstridKery** [comment]
Hey @schittarath3-rti ! I missed when this task was handed out. Which DB errors are we specifically worried about catching here? Is there a Jira issue?

**Comment by schittarath3-rti** [comment]
It catches an incorrect password error, but because there isn't one. It catches a generic account verification error. EWDCODE-119

File: backend/django/core/utils/utils_external_db.py [hunk 1]
```diff
@@ -25,6 +26,24 @@ def get_connection(db_type, connection_dict):
         raise ValueError(f"ERROR: database type {db_type} is invalid")
 
 
+def test_login(db_type, engine_database):
+    """Attempts to login with given connection."""
+    try:
+        # Perform a no-op
+        pd.read_sql("NULL", con=engine_database)
```
Comment (AstridKery) [inline backend/django/core/utils/utils_external_db.py:33]: Getting this error when I put in the correct password
Screen Shot 2022-07-18 at 3 48 57 PM

**Review by AstridKery — APPROVED** [review]
This is working for me now. Good work!

**Pull request merged**